scripts/test.sh

```diff
@@ -40,6 +43,7 @@ if [ -z "$CIRCLE_NODE_INDEX" ]; then
   CIRCLE_NODE_INDEX=1 run
   CIRCLE_NODE_INDEX=2 run
   CIRCLE_NODE_INDEX=3 run
+  CIRCLE_NODE_INDEX=4 run
```

Comment:
Is this going to require that we have another container when we run this on circleCI? If so we might want to put it in with another flow since we don't have too many jest tests yet.